x-pack/test/functional/apps/maps/group4/lens/choropleth_chart.ts

```diff
@@ -13,9 +13,13 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const filterBar = getService('filterBar');
 
-  // Failing: See https://github.com/elastic/kibana/issues/154065
-  // Failing: See https://github.com/elastic/kibana/issues/154064
-  describe.skip('choropleth chart', () => {
+  // Test requires access to Elastic Maps Service
+  // Do not skip test if failure is "Test requires access to Elastic Maps Service (EMS). EMS is not available"
+  describe('choropleth chart', () => {
+    before('', async () => {
+      await PageObjects.maps.expectEmsToBeAvailable();
+    });
+
```

Comment:
I believe this will still throw a test failure if EMS is unreachable. So it may break CI on some PRs and trigger some flaky test failure issues in Kibana. 

I wonder if the `expectEmsToBeAvailable` method could return a boolean and then we could [conditionally skip the test at runtime](https://mochajs.org/#inclusive-tests)? Something like:

```js
before('', async function () {
  if (!(await PageObjects.maps.expectEmsToBeAvailable())) {
    this.skip();
  }
});
```

Note, we can't use an arrow function in this example because it [changes the context of `this`](https://mochajs.org/#arrow-functions).

Reply:
I'm reconsidering my suggestion above. We should not be skipping tests due to network issues. I think it is more important to highlight any network connectivity failures in CI. We rely on network connectivity for our tests failing the test is the best option.